Assignment_1/Assignment_1/main.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  main.swift
+//  Assignment_1
+//
+//  Created by Brendon Cecilio on 10/1/19.
+//  Copyright © 2019 Brendon Cecilio. All rights reserved.
+//
+
+import Foundation
+import GameplayKit
+
+print("Welcome to my knock off Dungeons and Dragons Game 🐲!")
+print()
+sleep(4)
+
+print("Choose your weapon! (Axe/Staff/Bow and Arrow)")
+let weapon = readLine()?.lowercased() ?? ""
+print()
+
+switch weapon {
+case "axe":
+    sleep(1)
+    print("You are a warrior! ⚔️ You are an young orc about to enter your first Mak'gora, a duel of honor. Frightened and full of adrenaline you stare at your opponent intensely as you continue to sharpen your axe. Suddenly your enemy leaps to strike you!")
+case "staff":
+    sleep(1)
+    print("You are a mage! 🔮 While on a research expedition you and your ranging party come across an artifact. It is an ancient object from a different time, or different world perhaps. As you gaze upon it's glowing aura you are overwhelmed with a sense of greed and paranoia, urging you to take the artifact. As you reach for the glimmering orb a cave troll appears")
+case "bow and arrow":
+    sleep(1)
+    print("You are a hunter! 🏹 During your a hunting expedition you find a wild boar! After successfully killing the beast you and your hunting party capture your game and proceed to venture back to your village. The ground begins to shake and a thundering roar pierces your ear drums as you realize you've hunted someone else's game. A giant cyclops 👁 breaks through the trees and run towards you and your party...")
+default:
+
+    sleep(3)
+    print("\(weapon) was not an option 😡")
```

Comment:
You can use a while loop to give them an opportunity to choose a weapon again if they put in the wrong answer